examples/fps_cur_ridgecv_pipeline.py

```diff
@@ -0,0 +1,32 @@
+# This example shows how to use an FPS, then CUR seleuutor
```

Comment:
What is the reason here fo mixing FPS and CUR? Why we have 2 examples - one only with FPS and one with FPS and CUR?

Reply:
So I added this one because often the feature space is large, and you want to do FPS to remove redundancy but CUR to truly find a compact set of features.

Reply:
May we have also example for CUR? For users who are not very knowledgeable about the topic. Or is it redundant?

Reply:
It's a little redundant, as sending through CUR is essentially the same as sending through FPS. I just wanted to highlight stacking them versus sending through one.

Reply:
Ok, fine, I agree!